Developer-Guide.md

```diff
@@ -688,3 +691,23 @@ file exists and contains details of the image and how it was created:
 ```
 $ cat /var/lib/osbuilder/osbuilder.yaml
 ```
+
+## Running standalone
+
+It is possible to start the runtime without a container manager. This is
+mostly useful for testing and debugging purposes.
+
+### Create an OCI bundle
+
+```
+$ bundle="/tmp/bundle"
+$ rootfs="$bundle/rootfs"
+$ mkdir -p "$rootfs" && (cd "$bundle" && sudo kata-runtime spec)
```

Comment:
strictly speaking, I don't think you need the sudo on the `spec` command - but, it probably keeps all the file perms in sync?

Reply:
Good catch - fixed.